### PR TITLE
feat(Card): create web component

### PR DESCRIPTION
**Pull request by ariellalgilmore**
### Related Ticket(s)

#3534 

### Description

create `Card` web component

### Changelog

**New**

- add stories and test

**Changed**

- styles

### CODE REVIEW

File: packages/web-components/src/components/card/__stories__/card.stories.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+export const Default = ({ parameters }) => {
+  const { image, href, alt, defaultSrc, eyebrow, heading, copy, inverse, footer } = parameters?.props?.Card ?? {};
+  return html`
+    <dds-card color-scheme=${inverse ? 'inverse' : ''} href=${ifNonNull(href || undefined)}>
+      ${image
+        ? html`
+            <dds-image slot="image" alt="${ifNonNull(alt)}" default-src="${ifNonNull(defaultSrc)}"></dds-image>
+          `
+        : html``}
```
Comment (asudoh) [inline packages/web-components/src/components/card/__stories__/card.stories.ts:29]: Given there is no variable part in the template here:

```suggestion
        : ''}
```

An alternative is using `nothing` from `lit-html`: https://lit-html.polymer-project.org/guide/writing-templates#rendering-nothing

File: packages/web-components/src/components/card/__stories__/card.stories.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      <slot slot="eyebrow">${eyebrow}</slot>
+      <slot slot="heading">${heading}</slot>
+      ${copy
+        ? html`
+            <p>${copy}</p>
+          `
+        : html``}
```
Comment (asudoh) [inline packages/web-components/src/components/card/__stories__/card.stories.ts:36]: Given there is no variable part in the template here:

```suggestion
        : ''}
```

An alternative is using `nothing` from `lit-html`: https://lit-html.polymer-project.org/guide/writing-templates#rendering-nothing

**Review by asudoh — APPROVED** [review]
LGTM 👍 - Thanks @ariellalgilmore!

**Comment by jeffchew** [comment]
Hey @ariellalgilmore, is this the right alignment for the CardLink CTA icon? Looks like it's shifted to the right a bit:

Screen Shot 2020-09-08 at 3 21 40 PM

**Comment by ariellalgilmore** [comment]
Hey! @jeffchew yeah i'm hitting some styling issues now with card-link, but hopefully should have them fixed soon.

**Comment by ariellalgilmore** [comment]
created new branch because had some git issues merging

**Pull request closed without merging**